cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -72,8 +72,8 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION          ${CUML_MIN_VERSION_raft}
-      FORK             rapidsai
-      PINNED_TAG       branch-${CUML_BRANCH_VERSION_raft}
+      FORK             jinsolp
+      PINNED_TAG       batchnnd-dedup
```

Comment:
this needs to be changed after checking that the tests are working with the raft fix.